portfolio/src/main/webapp/style.css

```diff
@@ -7,6 +7,8 @@ c2dfe3
 */
 #content {
   width: 100%;
+}
+html{
```

Comment:
Please add a space before the {

See https://google.github.io/styleguide/htmlcssguide.html#Declaration_Block_Separation